homework_2/task_2.py

```diff
@@ -0,0 +1,54 @@
+# Напишите программу, которая принимает две строки вида “a/b” - дробь с числителем и знаменателем.
+# Программа должна возвращать сумму и произведение* дробей. Для проверки своего кода используйте модуль fractions.
+from fractions import Fraction
+
+input_fractions = input('Введите две дроби через пробел и знак + или * в формате - a/b c/d + ').split()
+
+fraction_1, fraction_2, signs = map(str, input_fractions)
```

Comment:
Тут же и так `str` будет
```suggestion
fraction_1, fraction_2, signs = input('Введите две дроби через пробел и знак + или * в формате - a/b c/d + ').split()
```

Reply:
Понятно, что пишу лишние строки кода.